src/app/user-set-card/user-set-card.component.scss

```diff
@@ -0,0 +1,49 @@
+@use '@angular/material' as mat;
+@use '@angular/material-experimental' as matx;
+
+@mixin theme($theme) {
```

Comment:
could you put only the styles that use the in this file? Move the mixins into a new file called `_user-set-card-theme.scss`